core/chains/evm/txmgr/txmgr.go

```diff
@@ -96,7 +96,7 @@ type reset struct {
 
 type Txm struct {
 	utils.StartStopOnce
-
+	orm              ORM
```

Comment:
add orm to be part of Txm struct so you use one for all components

Reply:
Definitely can't close on per component basis then. You'll need to pass parent contexts on every query that you would want to exit early on close.